constrants/Headerconstraint.js

```diff
@@ -0,0 +1,40 @@
+import ICONS from "../components/headericons";
+import ROUTES from "../routes/routes";
+import headerTheme from "../Theme/HeaderTheme";
+
+const CONSTANTS = {
+  navbarPages: [
+    {
+      text: 'Missions',
+      link: ROUTES.MISSIONS,
+      icon: ICONS.rocketicon,
+    },
+    {
+      text: 'Leaderboard',
+      link: ROUTES.LEADERBOARD,
+      icon: ICONS.trophyicon,
+    },
+    {
+      text: 'Rewards',
+      link: ROUTES.REWARDS,
+      icon: ICONS.rewardsicon,
+    }
+  ],
+
+  buttons: [
```

Comment:
As for these buttons you don't to define an array to iterate over it or export it from this file since there are only two items. You can just directly add this data to the component instances where you will use this